app/styles/main.scss

```diff
@@ -46,30 +53,57 @@ color: black;
   center {
     background-color: $black;
     color: $white;
-    }
+  }
 }
 
-h2 {
-margin: 0;
-color: #666;
-padding-top: 90px;
-font-size: 52px;
-font-family: "trebuchet ms", sans-serif;
+#form-spacing {
+  background-color: #fff;
+  background-image: none;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+  color: #555;
+  display: block;
+  width: 50%;
+  height: 34px;
+  padding: 6px 12px;
+  font-size: 14px;
+  line-height: 1.42857143;
+  -o-transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
+  transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
+  -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+  -webkit-transition: border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;
 }
 
-.item {
-background: #333;
-text-align: center;
-height: 300px !important;
+#form-spacing2 {
+  display: block;
```

Comment:
Properties should be ordered background-color, background-image, border, border-radius, -webkit-box-shadow, box-shadow, color, display, font-size, line-height, padding, -o-transition, -webkit-transition, transition